inputs/TransactionView.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { Text, View, ScrollView, SafeAreaView } from 'react-native';
+import styles from '../styles';
+import TransactionForm from './TransactionForm';
+
+interface Transaction {
+    id: number;
+    description: string;
+    amount: number;
+    date: Date;
+}
+
+export default function TransactionView() {
+    // Sample data for past transactions
+    const [transactions, setTransactions] = useState<Transaction[]>([
```

Comment:
This is adding bunch of dummy data, i guess for testing purposes? Otherwise every app should start with a blank state because when user installs it for the very first time, they havent purchased anything or interacted with anything yet